debian/control

```diff
@@ -46,6 +46,15 @@ Build-Depends: autoconf,
                python-virtualenv,
                uuid-dev,
                uuid-runtime,
+	       cryptsetup-bin | cryptsetup,
+	       gdisk,
+               parted,
+               python,
+               python-argparse,
+               sdparm | hdparm,
+               uuid-runtime,
```

Comment:
On Thu, 2015-01-29 at 06:22 -0800, Loic Dachary wrote:
> In debian/control:
> 
> > @@ -46,6 +46,15 @@ Build-Depends: autoconf,
> >                 python-virtualenv,
> >                 uuid-dev,
> >                 uuid-runtime,
> > +	       cryptsetup-bin | cryptsetup,
> > +	       gdisk,
> > +               parted,
> > +               python,
> > +               python-argparse,
> > +               sdparm | hdparm,
> > +               uuid-runtime,
> 
> already listed above

What we really need is a better way to say that all the ceph runtime
deps are build-time deps due to running make test at build time.  I
guess that's not possible without pre-processing the control/spec file,
but it felt wrong, so I just copied the whole lump in with some
frustration.

Naturally I'll tidy that up today.

Andrew Bartlett

-- 
Andrew Bartlett                       http://samba.org/~abartlet/
Authentication Developer, Samba Team  http://samba.org
Samba Developer, Catalyst IT          http://catalyst.net.nz/services/samba